shiny/ui/_html_deps_py_shiny.py

```diff
@@ -52,3 +53,24 @@ def spin_dependency() -> HTMLDependency:
         source={"package": "shiny", "subdir": "www/shared/py-shiny/spin"},
         stylesheet={"href": "spin.css"},
     )
+
+
+def busy_indicators_dependency() -> HTMLDependency:
+    return HTMLDependency(
+        "shiny-busy-indicators",
+        __version__,
+        source={"package": "shiny", "subdir": "www/shared/py-shiny/busy-indicators"},
+        stylesheet={"href": "busy-indicators.css"},
+        head=TagList(
+            # Enable busy indicators by default.
+            busy_indicators.use(),
```

Comment:
Could explore having something like a `html_attrs()` built on MetaDataNode that gets baked into htmltools